SetUp/autoDefine/autoDefine.py

```diff
@@ -321,7 +325,10 @@ def inputBuilder( directories, options, keep_going, save_intermed ):
             help='Continue iterating through directories even if define fails.')
     parser.add_argument('-i','--save_intermed', action='store_true',
             help='Do not remove files generated while setting up control')
+    parser.add_argument('-c','--cosmo', default=False, action='store_true',
```

Comment:
Don't make the user specify this on the command line, better to centralize calculation specifications